### PR TITLE
Ben/gpt2 parity

### PR DESCRIPTION
**Pull request by madisonmay**

### CODE REVIEW

File: tests/test_sequence.py [hunk 1]
```diff
@@ -220,7 +226,7 @@ def test_reasonable_predictions(self):
         self.assertTrue(any(pred["text"].strip() == "dog" for pred in predictions[0]))
 
     def test_chunk_long_sequences(self):
-        test_sequence = ["I am a dog. A dog that's incredibly bright. I can talk, read, and write!" * 10]
+        test_sequence = ["I am a dog. A dog that's incredibly bright. I can talk, read, and write! " * 10]
```
Comment (benleetownsend) [inline tests/test_sequence.py]: Im intrigued whether this space is required and if so why?

Comment (madisonmay) [reply tests/test_sequence.py]: I don't think it's required I think I just noticed that it was going to smush words together.  Let me confirm test still passes w/o space.

File: tests/test_utils.py [hunk 2]
```diff
@@ -49,7 +49,7 @@ def test_overlapping(self):
     def test_overlapping_gpt2(self):
         raw = ["Indico Is the best hey"]
         finetunex = [
-            ["Indico ", "Is the", " best", " hey"]
+            ["Indico", " Is the", " best", " hey"]
```
Comment (benleetownsend) [inline tests/test_utils.py]: Is this consistent on gpt-2 that the spaces always occur at the beginning? Can we somehow force this consistency in gpt-1 without effecting anything else?

File: tests/test_sequence.py (hunk 1 above)
Comment (benleetownsend) [reply tests/test_sequence.py]: Oh totally, I missed the position of the * 10 there.

**Pull request merged**